detector/other/SServices00.cpp

```diff
@@ -76,237 +76,254 @@ static Ref_t create_element(Detector& theDetector, xml_h element, Ref_t)  {
 //====================================================================
   cout << "\nBuilding SServices00"<< endl;
 
-  //==================================================
-  //           BuildTPCEndplateServices
-  //==================================================
-  
-  // start to prepare the Material and geometry as Mokka
+  bool build_tpc_cooling  = dd4hep::getAttrOrDefault(x_det, _Unicode(build_tpc_cooling),   bool(true) );
```

Comment:
This seems to be unused below. Is a condition missing, or is that intentional?

Reply:
you missed it: it's used in line 87 to control whether to build the TPC-related services.

Reply:
Ah yes. It was hiding very well ;) Sorry for the noise.